packages/amplify_datastore/android/src/test/kotlin/com/amazonaws/amplify/amplify_datastore/SchemaData.kt

```diff
@@ -19,661 +19,664 @@ import com.amplifyframework.core.model.*
 import com.amplifyframework.core.model.temporal.Temporal
 
 val postSchema = ModelSchema.builder()
```

Comment:
### Do any of these need to be updated with indexes? Or should we add one with indexes?

Reply:
The test model `Post` has a SDI defined, the index info has been added to this testing model and its schema, see line 24 - 26. And it's asserted by unit tests. iOS impl. has similar changes too.